dependabot_file.py

```diff
@@ -305,3 +315,15 @@ def build_dependabot_file(repo, group_dependencies, exempt_ecosystems) -> str |
     if compatible_package_manager_found:
         return dependabot_file
     return None
+
+
+def add_existing_ecosystem_to_exempt_list(exempt_ecosystems, existing_config):
+    """
+    Add the existing package ecosystems found in the dependabot.yml
+    to the exempt list so we don't get duplicate entries and maintain configuration settings
+    """
+    if existing_config:
+        existing_config_obj = yaml.safe_load(existing_config.decoded)
+        if existing_config_obj:
+            for entry in existing_config_obj.get("updates", []):
+                exempt_ecosystems.append(entry["package-ecosystem"])
```

Comment:
Is it okay to modify the `exempt_ecosystems` list? Seemed convenient, but not sure if creating a copy would have been better so as to preserve the original input? Overthinking or good idea?

Reply:
If we don't need the original, I'm good overwriting.